frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -115,26 +117,41 @@ class TableInteractive extends Component {
   static defaultProps = {
     isPivoted: false,
     hasMetadataPopovers: true,
-    renderTableHeaderWrapper: children => (
-      <div className={TableS.cellData} data-testid="cell-data">
-        {children}
-      </div>
-    ),
-    renderTableCellWrapper: children => {
-      const hasChildren = children != null && children !== "";
+    renderTableHeaderWrapper: (
+      children,
+      column,
+      columnIndex,
+      { theme } = {},
```

Comment:
Good catch - I recalled the `column` and `columnIndex` was there from the start, but because of a change from master it was no longer used.

I was planning on getting the header theming to work in this PR, but I decided to work on this later as there are subtle CSS issues that would take time to tackle.